swap/src/bin/swap_cli.rs

```diff
@@ -117,12 +117,15 @@ async fn main() -> Result<()> {
                 }
 
                 debug!("Received {}", bitcoin_wallet.balance().await?);
+            } else {
+                debug!(
+                    "Still got {} left in wallet, swapping ...",
+                    bitcoin_wallet.balance().await?
```

Comment:
It can really only happen if the user transfers more money to the same address he used previously without invoking the tool inbetween.

Should we actually swap the remaining funds then? I don't know :)
Starting the tool again is IMO a signal that the user wants to swap. Do they maybe want to add more funds before doing that? Potentially but I wouldn't bother implementing that for now.